### PR TITLE
Client Package actions scenarios automated

### PR DESCRIPTION
**Pull request by jyejare**
Two Scenarios:
1. Package installation on preupgrade client after upgrade
2. Package installation on new upgrade client after upgrade

### CODE REVIEW

File: upgrade/helpers/docker.py [hunk 1]
```diff
@@ -30,10 +30,7 @@ def generate_satellite_docker_clients_on_rhevm(
             'Clients count to generate on Docker should be atleast 1 !')
         sys.exit(1)
     satellite_hostname = os.environ.get('RHEV_SAT_HOST')
-    if ak:
-        ak = ak
-    else:
-        ak = os.environ.get('RHEV_CLIENT_AK_{}'.format(client_os.upper()))
+    ak = ak or os.environ.get('RHEV_CLIENT_AK_{}'.format(client_os.upper()))
```
Comment (san7ket) [inline upgrade/helpers/docker.py]: 👍

**Comment by jyejare** [comment]
@san7ket , Bhai ACK to de do ... sirf thenga dikhake chale gaye :)

**Comment by sghai** [comment]
@jyejare : Please correct the typo in commit message and I can merge.

**Comment by jyejare** [comment]
@sghai , Updated the same. Please revisit.

**Pull request merged**